barretenberg/cpp/src/barretenberg/dsl/acir_format/poseidon2_constraint.test.cpp

```diff
@@ -33,35 +34,39 @@ TEST_F(Poseidon2Tests, TestPoseidon2Permutation)
             .len = 4,
         };
 
-    AcirFormat constraint_system{ .varnum = 9,
-                                  .recursive = false,
-                                  .num_acir_opcodes = 1,
-                                  .public_inputs = {},
-                                  .logic_constraints = {},
-                                  .range_constraints = {},
-                                  .aes128_constraints = {},
-                                  .sha256_constraints = {},
-                                  .sha256_compression = {},
-                                  .schnorr_constraints = {},
-                                  .ecdsa_k1_constraints = {},
-                                  .ecdsa_r1_constraints = {},
-                                  .blake2s_constraints = {},
-                                  .blake3_constraints = {},
-                                  .keccak_constraints = {},
-                                  .keccak_permutations = {},
-                                  .pedersen_constraints = {},
-                                  .pedersen_hash_constraints = {},
-                                  .poseidon2_constraints = { poseidon2_constraint },
-                                  .multi_scalar_mul_constraints = {},
-                                  .ec_add_constraints = {},
-                                  .recursion_constraints = {},
-                                  .honk_recursion_constraints = {},
-                                  .bigint_from_le_bytes_constraints = {},
-                                  .bigint_to_le_bytes_constraints = {},
-                                  .bigint_operations = {},
-                                  .poly_triple_constraints = {},
-                                  .quad_constraints = {},
-                                  .block_constraints = {} };
+    AcirFormat constraint_system{
+        .varnum = 9,
+        .recursive = false,
+        .num_acir_opcodes = 1,
+        .public_inputs = {},
```

Comment:
I'd see if you can avoid all the `= {}`, they will be default constructed. (same in other files)

Reply:
This is just formatter changes, I'd prefer not generating such a huge diff in this PR :) Will try that in the newly added test though!

Reply:
oops!
```
    +preset-gcc *failed* | /usr/src/barretenberg/src/barretenberg/dsl/acir_format/honk_recursion_constraint.test.cpp:121:9: error: missing initializer for member 'acir_format::AcirFormat::gates_per_opcode' [-Werror=missing-field-initializers]
```